### PR TITLE
Sorting by multi-valued fields (#1445)

### PR DESCRIPTION
**Pull request by danielsotirhos**

### CODE REVIEW

**Comment by danielsotirhos** [comment]
Note that this branch was based off of `issues/danielsotirhos/sort-multi-value-fields`

File: test/service/test_response.py [hunk 1]
```diff
@@ -1420,6 +1420,27 @@ def test_ordering(self):
             self.assertEqual(hit_sort_values['asc'], sorted(hit_sort_values['asc']))
             self.assertEqual(hit_sort_values['desc'], sorted(hit_sort_values['desc'], reverse=True))
 
+    def test_multivalued_field_sorting(self):
+        """
+        Test that sorting by a multi-valued field responds with hits that are
+        correctly sorted based on the first value from each multi-valued field, and
+        that each multi-valued field itself is sorted low to high regardless of the search sort
```
Comment (hannes-ucsc) [inline test/service/test_response.py]: Is the statement in the last line true?

```suggestion
        that each multi-valued field itself is sorted low to high regardless of the search sort.
```

Comment (danielsotirhos) [reply test/service/test_response.py]: Yes. This test uses the `laboratory` field to verify 2 aspects of sorting:
- A request made using `sort` and `order` parameters against a multi-valued field like `laboratory` will order the hits based on the first (smallest) element from each hit's `laboratory` field. 
```
1442                self.assertEqual(laboratories, sorted(laboratories, reverse=reverse))
```
- Multi-valued fields (only `laboratory` verified here) is itself sorted low to high
```
1439                    self.assertEqual(laboratory, sorted(laboratory))
```

Comment (hannes-ucsc) [reply test/service/test_response.py]: I see, thank you!

**Comment by hannes-ucsc** [comment]
Not reviewing this right now since I requested changes on the base PR.

**Pull request merged**